### PR TITLE
add timber

### PR DESCRIPTION
**Pull request by pawalt**

### CODE REVIEW

File: monitoring/init.sh [hunk 1]
```diff
@@ -1,9 +1,11 @@
 #!/bin/bash
 
+# Timber also requires secrets. Right now, they exist in vault and are manually copied to monitoring, but once I add
+# cronjobs to icarus, we'll just sync to all namespaces.
```
Comment (ArmaanT) [inline monitoring/init.sh]: This can be done now, right?

File: monitoring/dashboard/values.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+image:
```
Comment (ArmaanT) [inline monitoring/dashboard/values.yaml]: This should be in a different PR, right?

File: monitoring/init.sh (hunk 1 above)
Comment (pawalt) [reply monitoring/init.sh]: No. Timber is  in the monitoring NS

File: monitoring/dashboard/values.yaml (hunk 2 above)
Comment (pawalt) [reply monitoring/dashboard/values.yaml]: sure
